test-data/unit/lib-stub/collections.pyi

```diff
@@ -1 +1,4 @@
 namedtuple = object()
+
+class OrderedDict:
```

Comment:
This should be generic for more realism, as the actual stubs define `OrderedDict[KT, VT]`.